spec/spec_helper.rb

```diff
@@ -51,17 +51,17 @@
   #   # is tagged with `:focus`, all examples get run. RSpec also provides
   #   # aliases for `it`, `describe`, and `context` that include `:focus`
```

Comment:
### Optional
- [ ] It would be better to make all the request test pass, we have 12 failing test cases. Although it is not a part of our task now, I highly recommend you to fix the test cases or comment them out if they are not part of the assignment
![Screenshot from 2022-04-06 23-57-54](https://user-images.githubusercontent.com/46388113/162093815-4365fb0e-2424-479e-b641-313c8298467d.png)